web/pdf_viewer.css

```diff
@@ -157,10 +157,13 @@
   right: 0;
   bottom: 0;
   background: url("images/loading-icon.gif") center no-repeat;
+  visibility: visible;
+  transition-property: visibility;
```

Comment:
Any reason that we couldn't just use the existing `background`-property instead, rather than introducing `visibility` here?

In any case, a good comment explaining what's happening here seems necessary to aid future readers (that might even be us in a few months time).

Reply:
I tried different things:
 - transition on `background` leads to finally transition on the position and not the image.
 - transition on `background-image` doesn't work.

We could make the transition on background-size (like for the other one), but in this case making the transition on the visibility avoid to have to know the dimensions and I thought it makes senses to change the visibility since we're toggling `notVisible`.